light-client/src/consensus/consensus_client.rs

```diff
@@ -44,15 +43,15 @@ struct LightClientStore {
     current_max_active_participants: u64,
 }
 
-impl<R: ConsensusRpc> ConsensusClient<R> {
+impl<R: ConsensusRpc> ConsensusLightClient<R> {
     pub fn new(
         rpc: &str,
         checkpoint_block_root: &[u8],
         config: Arc<Config>,
-    ) -> Result<ConsensusClient<R>> {
+    ) -> Result<ConsensusLightClient<R>> {
```

Comment:
*nitpick* but it seems like we can just access the `rpc` value from the `Config` directly rather than requiring to pass it in as a separate argument

Reply:
I see some benefits in leaving this as a separate argument, will merge it as it is for now.